docs/modules/ROOT/pages/cops_layout.adoc

```diff
@@ -7124,3 +7191,181 @@ RUBY
 === References
 
 * https://rubystyle.guide#no-trailing-whitespace
+
+== AllowMultilineFinalElement
+
+`AllowMultilineFinalElement` is a boolean option (`false` by default) that
+is available on the following Layout cops:
+
+  * FirstArrayElementLineBreak
+  * FirstHashElementLineBreak
+  * FirstMethodArgumentLineBreak
+  * FirstMethodParameterLineBreak
+  * MultilineArrayLineBreaks
+  * MultilineHashKeyLineBreaks
+  * MultilineMethodArgumentLineBreaks
+  * MultilineMethodParameterLineBreaks
+
+Those cops ignores their respective statements if all or the elements of
+the statement are on the same line. If `AllowMultilineFinalElement` is
```

Comment:
I think that here "expression" is often a better term than "statement", so I'd probably use it.